server/config/routes.js

```diff
@@ -360,6 +361,13 @@ exports.configure = function (app) {
   app.post('/cashboxes/:id/currencies', cashboxes.currencies.create);
   app.put('/cashboxes/:id/currencies/:currencyId', cashboxes.currencies.update);
 
+  // cash (aux/primary)
+  app.get('/cash', cash.list);
+  app.get('/cash/:uuid', cash.details);
+  app.post('/cash', cash.create);
+  app.put('/cash/:uuid', cash.update);
+  app.delete('/cash/:uuid', cash.debitNote);
```

Comment:
In you PR description you said there is no delete route for cash, but why you keep it here?